### PR TITLE
Add LF Edge reference in docs footer

### LINKED ISSUE #525: Documentation📄: Add LF Edge reference to documentation site
### What is the current documentation state?

Add a new line in the footer file reading:

Open Horizon is a [Stage Two project](https://wiki.lfedge.org/display/LE/Stage+2%3A+Growth) at [LF Edge](https://www.lfedge.org/).

This will need to be added to the `_config.yml` file at the end of the `footer_content` property value.  You will need to convert the above to HTML and backslash escape any quote marks.

### Where is this stated?

_No response_

### Why do you want to improve the statement?

_No response_

### Proposed Statement

_No response_

### Additional context.

_No response_

### PR DESCRIPTION
**Pull request by dotarjun**
## Description

Add LF Edge ref to the footer on the docs site

Fixes #525  

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [x] I have tagged the reviewers in a comment below incase my pull request is ready for a review
- [x] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

@joewxboy

### CODE REVIEW

**Review by johnwalicki — CHANGES_REQUESTED** [review]
Typo in line 1

**Comment by dotarjun** [comment]
Fixed

**Review by johnwalicki — APPROVED** [review]

**Review by johnwalicki — APPROVED** [review]
Lgtm

File: _config.yml [hunk 1]
```diff
@@ -138,7 +138,7 @@ nav_external_links:
 back_to_top: true
 back_to_top_text: "Back to top"
 
-footer_content: "Copyright &copy; 2015-2023 Varies per page, see headers. Documentation distributed by an <a href=\"https://github.com/open-horizon/open-horizon.github.io/blob/master/LICENSE\">CC-BY-4.0 license.</a>"
+footer_content: "Copyright &copy; 2015-2023 Varies per page, see headers. Documentation distributed by an <a href=\"https://github.com/open-horizon/open-horizon.github.io/blob/master/LICENSE\">CC-BY-4.0 license.</a> Open Horizon is a <a href= \https://wiki.lfedge.org/display/LE/Stage+2%3A+Growth\">Stage Two project</a> at <a href =\https://www.lfedge.org/\">LF Edge</a>."
```
Comment (joewxboy) [inline _config.yml:141]: It looks like the beginnings of your hyperlinks are missing quote marks:

You have: `<a href= \` in two places.

It should be `<a href=\"` instead in both places.

Can you please fix and update the PR?

**Review by joewxboy — CHANGES_REQUESTED** [review]
See the comment about line 141 in the `_config.yml` file.

File: _config.yml (hunk 1 above)
Comment (dotarjun) [reply _config.yml]: Done!

**Review by joewxboy — APPROVED** [review]

**Pull request merged**